### DISCUSSION TITLE
Authentication has never worked for me

### DISCUSSION THREAD

**Discussion by bgeery** [Tyrrrz/YoutubeDownloader#373] [Q&A]
I login with Google, and YTDownloader says I'm currently authenticated, but when it goes to process a playlist URL, I always get the same "Could not extract playlist title" error popup.  If I logout, things work as expected.  I've never had a version that authentication works any differently in YTDownloader.

**Comment by Tyrrrz** [comment]
Are you on latest version? I pushed an update yesterday that touches on that

**Reply by bgeery** [reply]
Yeah, latest version. That's why I decided to finally bring up the issue, to see if it's just me for some reason.

**Reply by Tyrrrz** [reply]
Do you, by chance, have multiple YouTube channels associated with your Google account?

**Reply by bgeery** [reply]
No, just the one.

**Reply by Tyrrrz** [reply]
Hmm, interesting. What kind of playlists does `Could not extract playlist title` show up on? Can you download private videos?

**Comment by bgeery** [comment]


**Reply by Tyrrrz** [reply]
Thanks. Can you also try to fetch a private playlist? For example the "Watch Later" playlist: https://www.youtube.com/playlist?list=WL

This will help figure out if the issue is specific to the playlist or to authentication in general.

**Reply by bgeery** [reply]
https://www.youtube.com/playlist?list=PLlFe2hF-YHMSXBCYg9j1C6ESeC8iID4zE is the playlist I mostly use with TYD. It's a public playlist, but I'm assuming authentication would allow downloading the few age-restricted videos in the playlist that TYD used to download, but stopped at some point.

**Reply by bgeery** [reply]
Let me try what you ask....standby.

**Reply by bgeery** [reply]
OK yes, https://www.youtube.com/playlist?list=WL works. [when authenticated]

**Reply by bgeery** [reply]
So do I just not understand the purpose of using authentication?  Is it only useful for processing private playlists?  I thought is would allow the downloading of age-restricted videos that YT requires to be logged in to view.

If I was wrong about the purpose of authentication, please accept my apology for wasting your time.

**Reply by Tyrrrz** [reply]
No, there just seems to be an issue that's specific to that playlist. You should be able to download it while authenticated. But clearly something is not working properly, so I'll try to investigate more when I have some time.

The main purpose of authentication is to access private content, mostly. But yes, it can also help with age-restricted videos and such (although they should generally still be downloadable, with very few exceptions).

**Reply by bgeery** [reply]
Well, at least you have a good sample playlist to test against now.  I'm glad is mostly just me. ;)

I'll check again if downloading age-restricted videos is still an issue.  It's been a few versions ago last time I tried.

**Reply by Tyrrrz** [reply]
Note that the workarounds for downloading age-restricted videos (without auth) is best-effort only. The only reason it still works sometimes is because YouTube left a few holes open in that regard, which I assume is temporary. So authentication may be the only solution for that in the future.

**Reply by bgeery** [reply]
Gotcha.  Thanks so much for the help, Tyrrrz.  If you need any further help or testing from me on this issue, just contact me and I'll be happy to help in any way I can.
